.github/workflows/cypress-pr.yml

```diff
@@ -65,11 +65,11 @@ jobs:
           record: true
           parallel: true
           group: ubuntu-regression
-          quiet: true
         env:
           # pass the Dashboard record key as an environment variable
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           CI: 'true'
           # Recommended: pass the GitHub token lets this action correctly
           # determine the unique run id necessary to re-run the checks
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DEBUG: 'cypress:*'
```

Comment:
do we really need all those logs in CI?

Reply:
> Enable debug logging so we can investigate our intermittent cypress failures.

This is only temporary until we get to the bottom of the failing Github Actions.